history-microservice/Dockerfile

```diff
@@ -34,7 +34,20 @@ ENV IDRINTH_API_BENCH_VERSION=$BUILD_VERSION
 ENV IDRINTH_API_BENCH_BUILT=$BUILD_TIME
 
 WORKDIR /project
-COPY --from=build /project/@idrinth/api-bench/history-microservice /project
+WORKDIR /api-bench/history-microservice
+RUN npm ci \
+    && npm run tsc \
+    && npm prune --production \
+    && (rm /api-bench/history-microservice/README.md || true) \
+    && (rm /api-bench/history-microservice/src/**/*.ts || true) \
+    && (rm /api-bench/history-microservice/src/**/*.ts || true) \
+    && clean-modules clean --yes \
+    && mkdir -p /project/@idrinth/api-bench/history-microservice \
+    && cp --recursive /api-bench/history-microservice/* /project/@idrinth/api-bench/history-microservice/ \
+    && cp /api-bench/history-microservice/.env /project/@idrinth/api-bench/history-microservice/ \
+    && git clone https://github.com/idrinth-api-bench/history-microservice
```

Comment:
this needs to happen first, otherwise the install and delete commands don't work